docs/DeveloperGuide.adoc

```diff
@@ -819,14 +830,14 @@ _{More to be added}_
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: Add category to person
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User searches for a specific client
+2.  AddressBook shows a list of users matching query
```

Comment:
Your product name is not addressbook anymore, right? So need to update it here (and all occurrences)